pages/form/edit/[formName].js

```diff
@@ -483,6 +491,42 @@ const FormEditor = () => {
                         },
                     ]}
                 />
+                <Card
+                    width={['500px', '700px', '800px']}
+                    marginBottom={`${theme.space[7]}px`}
+                    marginRight='auto'
+                    marginLeft='auto'
+                    display='flex'
+                    justifyContent='space-between'
```

Comment:
Doesn't display=flex align items horizontally by default ?

I thought you need to specify flex-direction=vertical for items to be aligned vertically.

If you delete the `justifyContent` property, does the UI still look the same ? I don't know if this property is needed.